src/swsssdk/scripts/sonic-db-cli

```diff
@@ -80,6 +81,8 @@ def execute_cmd(dbname, cmd, namespace, use_unix_socket=False):
             print()
         elif isinstance(resp, list):
             print("\n".join(resp))
+        elif isinstance(resp, dict):
+            print(json.dumps(resp))
```

Comment:
This changes the existing behavior. How should the existing code be adjusted?

Reply:
There are 3 test cases in sonic-mgmt which uses this command, i will change them accordingly.

Reply:
Not sure whether there are other components also referencing this CLI besides the regression test.
You may search for the "sonic-db-cli" in the whole "sonic-buildimage" repo (with all the submodules).
If there isn't any other code referencing this command, let's open the community PR and listen to the community.

Reply:
Thanks for reminding. I searched the code, there is indeed a few code uses this command. However, the only 3 places use this command with "HGETALL" are only example with pure string:

```
fsroot/var/lib/docker/overlay2/31f72acab53561006fd48985213a8e6a54c63002bd5a426cb77ea0a605ffeabd/diff/usr/local/bin/sonic-db-cli:Example 2: sonic-db-cli -n asic2 APPL_DB HGETALL VLAN_TABLE:Vlan10
fsroot/var/lib/docker/overlay2/06d3f1816216eba4658c7d4add84d45ddab468b03979c008cec652280136b117/diff/usr/local/bin/sonic-db-cli:Example 2: sonic-db-cli -n asic2 APPL_DB HGETALL VLAN_TABLE:Vlan10
fsroot/usr/local/bin/sonic-db-cli:Example 2: sonic-db-cli -n asic2 APPL_DB HGETALL VLAN_TABLE:Vlan10
src/sonic-py-swsssdk/build/scripts-2.7/sonic-db-cli:Example 2: sonic-db-cli -n asic2 APPL_DB HGETALL VLAN_TABLE:Vlan10
src/sonic-py-swsssdk/build/scripts-3.7/sonic-db-cli:Example 2: sonic-db-cli -n asic2 APPL_DB HGETALL VLAN_TABLE:Vlan10
src/sonic-py-swsssdk/src/swsssdk/scripts/sonic-db-cli:Example 2: sonic-db-cli -n asic2 APPL_DB HGETALL VLAN_TABLE:Vlan10
```

So, for now I suppose it is safe. Will listen to community.